### PR TITLE
Change download link to new wkhtmltopdf non broken link

### PR DESCRIPTION
**Pull request by praveenperera**
The original download website is broken, using new site

### CODE REVIEW

**Comment by Subtletree** [comment]
@mbriskar Need this merged 😄

**Comment by singhpratyush** [comment]
Until this is merged, you can use -
```sh
$ sudo dokku plugin:uninstall wkhtmltopdf
$ sudo dokku plugin:install https://github.com/praveenperera/dokku-wkhtmltopdf.git --committish change-download-link wkhtmltopdf
```

**Comment by mbriskar** [comment]
I am sorry,  I didn't expect others to use this. I am going to review PRs quickly

File: pre-build-buildpack [hunk 1]
```diff
@@ -4,12 +4,12 @@ set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
...
-export DEBIAN_FRONTEND=noninteractive
 apt-get -y  update 
-apt-get install -y wget xfonts-base xfonts-75dpi
-wget http://download.gna.org/wkhtmltopdf/0.12/0.12.2.1/wkhtmltox-0.12.2.1_linux-trusty-amd64.deb
-dpkg -i wkhtmltox-0.12.2.1_linux-trusty-amd64.deb
+apt-get install -y wget xfonts-base xfonts-75dpi libssl1.0-dev openssl
+cd /tmp && wget https://downloads.wkhtmltopdf.org/0.12/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz
+cd /tmp && tar xf wkhtmltox-0.12.4_linux-generic-amd64.tar.xz
+mv /tmp/wkhtmltox /opt/
+cd /usr/local/bin && ln -s /opt/wkhtmltox/bin/wkhtmltopdf && ln -s /opt/wkhtmltox/bin/wkhtmltoimage
```
Comment (mbriskar) [inline pre-build-buildpack:12]: do we have to use "cd"?

**Review by mbriskar — APPROVED** [review]

**Pull request merged**